notebooks/community/model_evaluation/automl_tabular_regression_model evaluation.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<p>Add text </p><p>You use a service account to create Vertex AI Pipeline jobs. If you do not want to use your project's Compute Engine service account, set <code>SERVICE_ACCOUNT</code> to another service account ID.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>